acct-group/scanner/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
Also I'm sorry but we can not add new packages without maintainer, it's against the policy. 
  https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0602

Didn't you originally want to become the proxy-maintainer for all of these packages?

Reply:
No, I didn't. Initially I just wanted to submit a version bump, as it increases the number of supported devices (including mine). Then @heroxbd suggested me to fix the inconsistencies using this case changes. Maintaining it is a little too much for me, especially given how long and difficult this process of adding simple version bump became.

Let's then leave it here in case of anyone would be interested.